docs/community.rst

```diff
@@ -1,4 +1,35 @@
 .. _community:
 
 Community
-=========
\ No newline at end of file
+=========
+
+.. _code_of_conduct:
+
+Code of conduct
+---------------
+
+This project has adopted the `GitHub community guidelines 
+<https://help.github.com/en/github/site-policy/github-community-guidelines>`_.
+
+.. _communication:
+
+Real-time communication
+-----------------------
+
+For real time communication please join us on
+`Gitter <https://gitter.im/fairlearn/community>`_.
+All are welcome!
+
+.. _developer_calls:
+
+Developer Calls
+---------------
+
+We have a monthly contributor meeting every second Thursday of the month at
+`3pm UTC <http://time.unitarium.com/utc/3pm>`_. All are welcome! Please reach
```

Comment:
Let's try to be more user-friendly with the way we communicate the time. Can we list the times in the three zones where we currently have participants? I like the idea of linking to a time-zone converter (or even including a widget), but if we do that, we should pick a provider with no advertising.

Reply:
I deliberately don't want to put times there because of daylight savings. As you may know daylight savings doesn't occur everywhere, and even for the places where it occurs it doesn't occur on the same day. Happy to take suggestions for a better converter :-)

Reply:
if you worry as far out as daylight savings then 3pm UTC is probably incorrect anyway (since we'll need to shift to reflect daylight savings). another idea would be to just say that we do it every second tuesday of the month and list a couple of next planned meetings with times?

Reply:
Yeah I assumed that we'll move it to accommodate, but I guess this works better.

Reply:
looks good.